readinglist/resource.py

```diff
@@ -93,6 +95,12 @@ class RessourceSchema(colander.MappingSchema):
     _id = colander.SchemaNode(colander.String(), missing=colander.drop)
     last_modified = TimeStamp()
 
+    __readonly_fields__ = ('_id', 'last_modified')
```

Comment:
Having an `Options` or `Meta` subclass seems also good to me, and it seems easy to have interchangeable subclasses (with different readonly_fields for instance, or which computes the fields only when accessed).